lib/Tool/Newsletter.php

```diff
@@ -115,6 +119,8 @@ public static function prepareMail(
 
         $mail->setBodyHtml($contentHTML);
         $mail->setBodyText($contentText);
+        // Adds the plain text part to the message, that it becomes a multipart email
+        $mail->addPart($contentText, 'text/plain');
```

Comment:
Why? See: 
https://github.com/pimcore/pimcore/blob/d688a67780b6ae733e145110bbbc04b3278e7190/lib/Mail.php#L567

Reply:
Line 567 is part of the send function, the newsletter is send via sendWithoutRendering function:

Pimcore::getEventDispatcher()->dispatch(DocumentEvents::NEWSLETTER_PRE_SEND, $event);
            $mail->sendWithoutRendering($mailer);
            Pimcore::getEventDispatcher()->dispatch(DocumentEvents::NEWSLETTER_POST_SEND, $event);

Reply:
Good point 👍😊